tests/dummy/app/pods/demo/template.hbs

```diff
@@ -3,63 +3,64 @@
       {{notification-message notification=notification}}
   {{/each}}
 </div>
+<div class='frost-text-demo'>
+  <h2>frost-text</h2>
 
-<h2>frost-text</h2>
+  <div id="content">
+    <div class="dummy-header">Basic</div>
+    <div style="padding: 10px 20px;">
```

Comment:
Would be nicer to use a class for this, just make it specific to the demo type e.g. `.ember-frost-text-demo .demo-padding`